06-Customizing AzTS for your org/Extending AzTS/ManagementGroupBasedComplianceInitiative.md

```diff
@@ -0,0 +1,102 @@
+
+![Resources](../../Images/06_MGCI_AddCOnfig_1.png)
+
+7. Add new application setting 'UIConfigurations__complianceInitiativeFeatureConfiguration__IsEnabled' with value as 'true'. Click 'OK'.
+
+![Resources](../../Images/06_MGCI_AddConfig_2.png)
+
+8. Add new application setting 'ComplianceInitiativeActionItem__0__PrivilegedEditorIds__0' with value as 'user object id'. Click 'OK'. This step needs to be done per user who needs access to set/update Management Group driven Compliance Initiative.(PrivilegedEditorIds is an array so should be incremented sequentially for further users being added.)
+
+![Resources](../../Images/06_MGCI_AddConfig3.png)
+
+> **Note:** User's object id can be referenced from Azure AD.
+
+9. Add new application setting 'ControlActionItem__0__PrivilegedEditorIds__0'  with value as 'user object id'. Click 'OK'. This step needs to be done per user who needs access to set/update Management Group driven Compliance Initiative(PrivilegedEditorIds is an array so should be incremented sequentially for further users being added.)
+![Resources](../../Images/06_MGCI_AddConfig_4.png)
+> **Note:** User's object id can be referenced from Azure AD.This step can be skipped if already completed as part of adding user as Privileged editor for CMET.
+
+10. Add new application setting 'ControlEditorFeatureConfiguration__IsBulkEditEnabled' with value as true. Click 'OK'. 
+
+![Resources](../../Images/06_MGCI_AddConfig_5.png)
+
+>Note: If you have already enabled [bulk edit feature for CMET](./FeaturesInCMET.md#bulk-edit), you can skip step 11 and 12.
+11. Add new application setting 'FeatureManagement__BulkEdit' with value as true. Click 'OK'.
+
+![Resources](../../Images/06_MGCI_AddConfig_6.png)
+
+12.Add new application setting 'UIConfigurations__ControlEditorFeatureConfiguration__IsBulkEditEnabled' with value as true. Click 'OK'.
+
+![Resources](../../Images/06_MGCI_AddConfig_7.png)
+
+13. Click 'OK' and then click 'Save'.
+
+![Resources](../../Images/06_MGCI_AddConfig_8.png)
+
+
+### 2.	Add Required configuration to Metadata Aggregator to enable MGTreeProcessor and PIM API support
+
+
+1. Go to Azure Portal.
+2. Navigate to AzTS host subscription -> AzTS host resource group.
+3. Go to AzTS Metadata Aggregator function app (AzSK-AzTS-MetadataAggregator-xxxxx).
+4. Go to Configurations under Settings.
+5. Click on ‘New application setting’ to add required app settings.
+6. Add new application setting 'FeatureManagement__ManagementGroups' with value as true. Click 'OK'. 
+
+![Resources](../../Images/06_MGCI_EnableMGTree.png)
+
+>Note: Please follow steps 7 and 8 in case of Single tenant AzTS setup. These can be skipped for Multi-Tenant AzTS setup.
+7. Click on 'New application setting' to add required app settings.
+8. Add new application setting 'ManagementGroupConfigurations__ManagementGroupId' with value as parent Management Grroup Id. Click 'OK'. 
```

Comment:
Is step 8, Which MG parent is excepted where there are multiple tenants onboarded?